model/Concept.php

```diff
@@ -508,7 +508,13 @@ public function getProperties()
                 $sprop = "<$prop>";
             }
 
-            if (!in_array($prop, $this->DELETED_PROPERTIES) || ($this->isGroup() === false && $prop === 'skos:member')) {
+            // delete notations unless configured to show them
```

Comment:
This block should be moved outside the foreach loop, i.e. around line 499. There is no need to create `$deleted` for every single property value; it can be done just once.

Reply:
It think it is fixed now

Reply:
Done